frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/ISimpleGraphTableProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.virtuslab.gitmachete.frontend.ui.api.table;
+
+import com.intellij.ui.table.JBTable;
+import org.checkerframework.checker.guieffect.qual.UIEffect;
+
+import com.virtuslab.gitmachete.backend.api.IGitMacheteRepositorySnapshot;
+
+public interface ISimpleGraphTableProvider {
```

Comment:
Hmm actually is there any reason for `Simple`? Maybe just `IGraphTableProvider`?